linalg/petsc.hpp

```diff
@@ -139,9 +141,12 @@ class PetscParVector : public Vector
    /// Returns the global number of rows
    PetscInt GlobalSize() const;
 
-   /// Conversion function to PETSc's Vec type
+   /// Typecasting to PETSc's Vec type
```

Comment:
The term "conversion function" comes from the c++ standard (12.3.2), see e.g. https://en.cppreference.com/w/cpp/language/cast_operator. No need to change it -- just FYI.